solr_conf/schema.xml

```diff
@@ -631,6 +631,7 @@ v         currently supported on types that are sorted internally as strings
    <field name="sourceResource_type_id" type="string" indexed="true" stored="true" multiValued="true" />
    <field name="sourceResource_type_type" type="string" indexed="false" stored="false" multiValued="true" />
    <field name="sourceResource_type_providedLabel" type="string" indexed="true" stored="true" multiValued="true" />
+   <field name="sourceResource_type_name" type="string" indexed="true" stored="true" multiValued="true" />
```

Comment:
Is the field `sourceResource.type.name` currently being indexed into Solr?  Because if not, I could use `sourceResource.type.providedLabel` or `sourceResource.type.id` for this PR.